sonar-ws/src/main/protobuf/ws-components.proto

```diff
@@ -128,3 +124,10 @@ message Component {
   }
 }
 
+message Organization {
```

Comment:
I have the feeling Organization should be in the ws-common.proto to have a common representation of an organization